source/features/update-pr-from-base-branch.tsx

```diff
@@ -145,7 +150,7 @@ async function init(signal: AbortSignal): Promise<false | void> {
 	delegate('.rgh-update-pr-from-base-branch', 'click', handler, {signal});
 	observe([
 		'.mergeability-details > *:last-child', // Old view - TODO: Drop after June 2025
-		'[class^="MergeBox-module__mergePartialContainer"]',
+		'[aria-label="Conflicts"] [class^="MergeBoxSectionHeader-module__wrapper"] h3 + .fgColor-muted',
```

Comment:
I like the idea but this is so overly specific 🥲

Reply:
> only part of the merge box was updated

via React, hence #6554 

Basically that issue is about views that update more precisely, leaving the DOM with `rgh-seen`  classes. A lot of it could be resolved by clearing those classes when the URL changes, but the not all.

Reply:
I see you copied the selector from `pr-base-commits`. Can you move it to selectors.ts instead? This means future fixes can affect both features.

Reply:
But actually we don't need this, you can target `MergeBoxSectionHeader`, it's already fully regenerated